src/main/java/teammates/common/datatransfer/FeedbackSessionResponseStatus.java

```diff
@@ -68,15 +68,19 @@ public FeedbackSessionResponseStatus() {
 
     /**
      * Returns list of students who did not respond to the feedback session.
```

Comment:
Final touch, can you remove the period at the end?

Reply:
I think the period is necessary for JavaDocs.

If removed checkstyle gives an error.

Reply:
@tshradheya take a look at the next line. @bqnguyen94 is giving a legitimate suggestion here.

Reply:
Ohh sorry 😅 My bad

Reply:
@mbi90 you haven't update the code here?

Reply:
I corrected the place of the period. I tried removing the period completely but that caused a failure (at gradle lint)

Reply:
Hmm that's strange. What's the error message?